csuntechlab/takeoff
# TKO-33 -- FE: Choose a JS library for tagging and implement it (on edit-profile page)

# Description
Used `choice.js` library for tagging academic interests.
  
# Testing
- Add academic interests on the '/profile-setup' view. Save profile and verify in Vue devtools that form.academicInterests has been updated.

- Navigate to `/profile` and see that the example tags are not editable

# Installation
`Yarn install` probably
